cmd/virt-tail/main.go

```diff
@@ -166,6 +157,11 @@ func (v *VirtTail) watchFS() error {
 					rerr := errors.New("socketFile got removed")
 					log.Log.V(3).Infof("watchFS error: %v", rerr)
 					return rerr
+				} else if event.Name == termFile {
```

Comment:
I would group the two events, they lead to the same outcome (socketFile and termFile deletion).

Reply:
on termFile we are going to exit cleanly, if hte socket is instead abrupted we are going to exit with an error.